storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -267,6 +271,11 @@ static struct my_option my_long_options[] = {
      &opt_s3_session_token, &opt_s3_session_token, 0, GET_STR_ALLOC,
      REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
 
+    {"s3-storage-class", OPT_S3_STORAGE_CLASS,
+     "S3 storage class. STANDARD|STANDARD_IA|GLACIER|...",
```

Comment:
Are these three dots necessary? ".." ? Do you mean that there are more options supported?

Reply:
Maybe not on Amazon S3 (I'm not sure), but there are other S3 implementations (such as [Ceph RadosGW](https://docs.ceph.com/en/latest/radosgw/) or [MinIO](https://min.io/)) that can define arbitrary storage classes. Since the option itself will accept any storage class name and pass it on to the API (as it should), perhaps it makes sense to use `...` to suggest there may be other possibilities.

Reply:
Thanks for the clarification!

@rahulmalik87 can you add some documentation about the "..". Please say that it meant for passing custom storage class names provided other S3 implementations such as MinIO CephRadosGW, etc.

Reply:
--s3-storage-class=name
                      S3 storage class. STANDARD|STANDARD_IA|GLACIER|...
                      ... is meant for passing custom storage class names
                      provided by other S3 implementations such as MinIO
                      CephRadosGW, etc.